src/pages/settings/Profile/Contacts/NewContactMethodPage.js

```diff
@@ -56,91 +54,109 @@ const defaultProps = {
     loginList: {},
 };
 
+const getPhoneLogin = (phoneOrEmail) => {
+    if (_.isEmpty(phoneOrEmail)) {
+        return '';
+    }
+
+    return LoginUtils.appendCountryCode(LoginUtils.getPhoneNumberWithoutSpecialChars(phoneOrEmail));
+};
+
+const validateNumber = (values) => {
+    const parsedPhoneNumber = parsePhoneNumber(values);
+
+    if (parsedPhoneNumber.possible) {
+        return parsedPhoneNumber.number.e164;
+    }
+
+    return '';
+};
+
```

Comment:
Since it's not used as an exported method, we can just place it inside the functional component.